server/rooms.ts

```diff
@@ -986,7 +986,12 @@ export class GlobalRoom extends BasicRoom {
 			}
 		}
 		const stack = stackLines.slice(0, 2).join(`<br />`);
-		const crashMessage = `|html|<div class="broadcast-red"><b>${crasher} has crashed:</b> ${stack}</div>`;
+		let crashMessage;
+		if (/private|chat-plugins&#x2f;_/.test(stack)) {
```

Comment:
`chat-plugins&#x2f;_`?

Reply:
Yeah, there is `Chat.escapeHTML` used before for whatever reason.

Reply:
Yes, but we have plenty of chat plugins that _aren't_ private code. All the chat plugins that _are_ private code will have `private` in their name.

Reply:
I would codify that somewhere in the plugin loader, and also maybe make sure something like "private-messages" wouldn't be parsed as a private plugin either.

Reply:
It's already codified by `.gitignore`. :|

Reply:
It's chat plugins ending in `-private`.